anomalib/data/avenue.py

```diff
@@ -198,7 +199,7 @@ def __init__(
         task: TaskType = TaskType.SEGMENTATION,
         image_size: Optional[Union[int, Tuple[int, int]]] = None,
         center_crop: Optional[Union[int, Tuple[int, int]]] = None,
-        normalize: bool = True,
+        normalization: Union[InputNormalizationMethod, str] = InputNormalizationMethod.IMAGENET,
```

Comment:
picky comment: could you define `InputNormalizationType = Union[InputNormalizationMethod, str]` somewhere?

to be consistent with 

```
FeatureExtractorParams = Union[TimmFeatureExtractorParams, TorchFXFeatureExtractorParams]
```

in https://github.com/openvinotoolkit/anomalib/pull/748

i'm actually wondering if this is necessary since `InputNormalizationMethod` inherits from `str` and `Enum`, but in any case it would be nice to avoid repetitive `Union[...]` for maintenance purposes

Reply:
> i'm actually wondering if this is necessary since InputNormalizationMethod inherits from str and Enum

This is a good point, actually. This works fine:
```
>>> isinstance(InputNormalizationMethod.IMAGENET, InputNormalizationMethod)
True
>>> isinstance(InputNormalizationMethod.IMAGENET, str)
True
```
I guess the only question is if we want to allow users to instantiate the class with a string instead of the enum. So, do we want to allow something like this:
```
mvtec = MVTec(..., normalize="imagenet", ...)
```

Reply:
> So, do we want to allow something like this:
> 
> ```
> mvtec = MVTec(..., normalize="imagenet", ...)
> ```

Unless there it makes handling things harder somehow, I'd say yes :)

Reply:
In that case we need to keep the `str` typing. For now we'll merge it as, so with the Union typing. We could consider switching to custom types later if needed.